langchain/callbacks/tracers/schemas.py

```diff
@@ -31,6 +31,18 @@ class TracerSession(TracerSessionBase):
     id: int
 
 
+class TracerSessionV2Base(TracerSessionBase):
```

Comment:
make this `TracerSessionV2Create` and it needs an optional UUID

Reply:
Are you saying the tenant ID needs to be optional or it should have the `id` key but it's optional here? I'm not sure when we'd want to have it be optional ?